Makefile

```
@@ -163,6 +163,13 @@ _opt_pylint:
 	@command -v flake8 && { $(MAKE) pylint; exit $$?; } \
 		|| echo "SKIP: pylint (flake8 not found)"
 
+commitlint:
+	$(realpath build/bin/nvim) -u NONE +"luafile scripts/lintcommit.lua"
```

is there a reason you are not using `-es` ? It will avoid starting the UI, which avoids weird jank.
```suggestion
	$(realpath build/bin/nvim) -es -u NONE +"luafile scripts/lintcommit.lua"
```

---

The reason was that I forgot it existed :). Thanks.